dsl/pace/dsl/dace/dace_config.py

```diff
@@ -170,6 +178,9 @@ def get_backend(self) -> str:
     def get_orchestrate(self) -> DaCeOrchestration:
         return self._orchestrate
 
+    def sync_debug(self) -> bool:
+        return dace.config.Config.get("compiler", "cuda", "syncdebug")
```

Comment:
Does this get the value, or set it? I can't tell.

Reply:
This gets it, I'll rename